test/dotnet-migrate.Tests/GivenThatIWantToMigrateTestApps.cs

```diff
@@ -297,6 +297,15 @@ private void Restore(string projectDirectory)
                 .Pass();
         }
 
+        private void Restore3(string projectDirectory)
+        {
+            new TestCommand("dotnet")
```

Comment:
I added this there as well. Check the rebase. I also added an actual Restore3Command test command.